packages/nextjs/mocks/msw/msw-enable.ts

```diff
@@ -0,0 +1,42 @@
+import { handlers } from "./graphql";
+import { groqHandlers } from "./groq/handlers";
+
+/**
+ * Importing this file enables MSW.  It works both client-side and server-side.
+ */
+
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+// See https://github.com/mswjs/interceptors/issues/159
+const mswUnsupported = process.version.startsWith("v18");
+if (mswUnsupported) {
+  throw new Error(`[MSW Mock Server] MSW does not support mocking 'fetch' in Node v18`);
```

Comment:
😱

Reply:
Yeah, right? And `urql` uses fetch by default.
Another option was to supply `node-fetch` to urql but I wasn't sure about the sanity client. I decided to just require Node 16 for now, until `msw` fixes this.

Reply:
Or maybe globally replacing `fetch` with `node-fetch` if the mocks are enabled.